### PR TITLE
In Which We Fix Our Version Of Tornado To Match Jupyter

### PR DESCRIPTION
**Pull request by waltaskew**
pip installing civis-jupyter-notebook currently results in an incompatible version pair between tornado and jupyter. Our version of jupyter requires tornado<6. Versions of jupyter after 5.7.5 have a fix for this: https://github.com/jupyter/notebook/pull/4449 

The resulting jupyter from pip installing this package will hang indefinitely attempting to connect to the kernel. This currently works in platform because the datasciene-python image specifies an older tornado version. However, the hanging kernel behaviour appears when trying to use other docker images.

@salilgupta1

### CODE REVIEW

**Comment by mheilman** [comment]
Just a note: this is indirectly related to https://civisanalytics.atlassian.net/browse/CIVP-18029

File: requirements.txt [hunk 1]
```diff
@@ -2,6 +2,9 @@ civis~=1.6
 requests~=2.18
 click~=6.7
 notebook~=5.1,!=5.7.*
+# for versions of jupyter before 5.7.5 we need to specify tornado<6
+# as these older versions of jupyter will break with tornado 6
+tornado<6
```
Comment (salilgupta1) [inline requirements.txt]: I think we should upgrade the notebook package to something like `notebook>=5.7.5,<6.0.0`

I can't remember why we do not want any `5.7` versions ... 

If we can successfully upgrade the notebook version to `5.7.x` then I think we can also fix the issue with upgrading python3 docker image to use datascience-python 5.x

Comment (mikesaelim) [reply requirements.txt]: #33

Comment (waltaskew) [reply requirements.txt]: I don't really have bandwidth to investigate the impact of changing our jupyter version. I was just trying to fix what we have currently.

**Comment by salilgupta1** [comment]
yea this is super annoying. Alright, I'm fine with this temporary fix and when jupyter release 5.8 we can upgrade to that and remove the tornado restriction.

**Pull request merged**